### PR TITLE
Hide `user-local-time` if timezone is unknown

### LINKED ISSUE #2612: `user-local-time` shows empty time for people with no commits
Maybe a nicer message will help.

Screenshot from 2019-12-10 19-52-58

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
   Fixes #2612 

2. TEST URLS:
   https://github.com/sindresorhus/refined-github/issues/3523 the first posts user

3. SCREENSHOT:
   None

### CODE REVIEW

File: source/features/user-local-time.tsx [hunk 1]
```diff
@@ -102,6 +103,14 @@ function init(): void {
 			return;
 		}
 
+		const timeout = delay(300);
+		// This will delay adding the icon for 300ms, enough to check if the date is store as "not found"
+		const race = await Promise.race([timeout, getLastCommitDate(login)]);
```
Comment (fregante) [inline source/features/user-local-time.tsx:108]: `getLastCommitDate(login)` is repeated below. I suggest this or `datePromise`

```suggestion
		const date = getLastCommitDate(login);
		const race = await Promise.race([timeout, date]);
```

Comment (yakov116) [reply source/features/user-local-time.tsx:108]:
```suggestion
		const datePromise = getLastCommitDate(login);
		const race = await Promise.race([timeout, datePromise]);
```

File: source/features/user-local-time.tsx [hunk 2]
```diff
@@ -125,7 +135,7 @@ function init(): void {
 
 		const date = await getLastCommitDate(login);
```
Comment (yakov116) [inline source/features/user-local-time.tsx:136]:
```suggestion
		const date = await datePromise;
```

**Comment by fregante** [comment]
🙌

**Pull request merged**